src/index.js

```diff
@@ -70,6 +71,7 @@ function readOrder() {
   };
 }
```

Comment:
Super nit-picky, but does it makes sense to have a:
```js
function readOrderFlags() {
  return {
    isLiquidityOrder: document.querySelector("#isLiquidityOrder").checked ? true : undefined,
  }
}

// ... Then below use:

{
  ...order,
  ...swapAmount,
  ...readOrderFlags(),
  from: await signer.getAddress(),
}
```

Just so any future flags we add just go into this one place.

Super nit-picky though - fine with how it is.